packages/stylelint/index.js

```diff
@@ -0,0 +1,41 @@
+const merge = require('deepmerge');
+const { join } = require('path');
+const StylelintPlugin = require('stylelint-webpack-plugin');
+const { lint, formatters } = require('stylelint');
+
+module.exports = (neutrino, opts = {}) => {
+  const options = merge({
+    pluginId: 'stylelint',
+    files: '**/*.+(css|scss|sass|less)',
+    context: neutrino.options.source,
+    failOnError: neutrino.options.command !== 'start',
+    quiet: neutrino.options.command === 'start',
```

Comment:
This ensures that the actual error is output, when `failOnError` is enabled.

E.g. with settings the way they currently are, you will get:
```
⠼ Building project
src/css/index.scss
 8:1  ✖  Unexpected empty line before at-rule   at-rule-empty-line-before


✖ Building project failed
Error: Failed because of a stylelint error.

    at linterSuccess (/Users/timkelty/Code/neutrino-dev/node_modules/stylelint-webpack-plugin/lib/run-compilation.js:34:14)
    at <anonymous>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

But if we don't set `quiet` (defaulting to true), you won't get the actual error:
```
✖ Building project failed
Error: Failed because of a stylelint error.

    at linterSuccess (/Users/timkelty/Code/neutrino-dev/node_modules/stylelint-webpack-plugin/lib/run-compilation.js:34:14)
    at <anonymous>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

The other option is to not set either `failOnError` _or_ `quiet`, since by default it emits errors to webpack, so the build will fail anyway.

Reply:
> The other option is to not set either `failOnError` or `quiet`, since by default it emits errors to webpack, so the build will fail anyway.

I opted to go this route.